### PR TITLE
Receiving translation for OIE

### PR DESCRIPTION
**Pull request by eng-prod-CI-bot-okta**
OIE translation for the latest commit: https://github.com/okta/okta-help/commits/docs_translations_OIE

### CODE REVIEW

File: oie/ja-jp/content/topics/api/trusted-origins-iframe.htm [hunk 1]
```diff
@@ -300,8 +300,8 @@ <h2>仕組み</h2>
...
 </p>
                                                     <p>開発者向けドキュメントについては、<a href="https://developer.okta.com/docs/reference/api/trusted-origins/">「Trusted Origins API」</a>を参照してください。</p>
-                                                    <h2>開始する前に</h2>
-                                                    <p>iFrameの埋め込みの信頼済みオリジンは、<span class="uicontrol">［Customizations（カスタマイズ）］</span>&gt;<span class="uicontrol">［Other（その他）］</span>&gt;<span class="uicontrol">［Enable iFrame Embedding（iFrameの埋め込みを有効化）］</span>の下のiFrame埋め込みが使用されていない場合にのみ機能します。最初に必要なすべての信頼済みオリジンを埋め込んでから、［Customizations（カスタマイズ）］オプションを有効にしている場合は無効にすることをお勧めします。これにより、既存のiFrameを誤って破損することがなくなります。
+                                                    <h2>はじめに</h2>
+                                                    <p>iFrameの埋め込みの信頼済みオリジンは、<span class="uicontrol">［Customizations（カスタマイズ）］</span>&gt;<span class="uicontrol">［その他］</span>&gt;<span class="uicontrol">［Enable iFrame Embedding（iFrameの埋め込みを有効化）］</span>の下のiFrame埋め込みが使用されていない場合にのみ機能します。最初に必要なすべての信頼済みオリジンを埋め込んでから、［Customizations（カスタマイズ）］オプションを有効にしている場合は無効にすることをお勧めします。これにより、既存のiFrameを誤って破損することがなくなります。
```
Comment (paulwallace-okta) [inline oie/ja-jp/content/topics/api/trusted-origins-iframe.htm:304]: This UI control label has EN removed
［Other（その他）］> ［その他］

File: oie/ja-jp/content/topics/api/trusted-origins-iframe.htm [hunk 2]
```diff
@@ -337,7 +337,7 @@ <h3><a name="1."></a>1. OktaサインインページまたはOktaリソースを
                                                         <li value="5">
                                                             <p>
                                                                 <p><span class="uicontrol">［Customizations（カスタマイズ）］</span>でiFrameの埋め込みを無効にします。 </p>
-                                                                <p><span class="uicontrol">Admin Console</span>の警告メッセージに表示されている<span class="uicontrol">iFrame埋め込み</span>リンクをクリックするか、<span class="uicontrol">［Customizations（カスタマイズ）］</span> &gt; <span class="uicontrol">［Other（その他）］</span> &gt; <span class="uicontrol">［iFrame Embedding（iFrame埋め込み）］</span>に移動し、<span class="uicontrol">［Enable iFrame embedding（iFrame埋め込みを有効にする）］</span>のチェックを外します。</p>
+                                                                <p><span class="uicontrol">Admin Console</span>の警告メッセージに表示されている<span class="uicontrol">iFrame埋め込み</span>リンクをクリックするか、<span class="uicontrol">［Customizations（カスタマイズ）］</span> &gt; <span class="uicontrol">［その他］</span> &gt; <span class="uicontrol">［iFrame Embedding（iFrame埋め込み）］</span>に移動し、<span class="uicontrol">［Enable iFrame embedding（iFrame埋め込みを有効にする）］</span>のチェックを外します。</p>
```
Comment (paulwallace-okta) [inline oie/ja-jp/content/topics/api/trusted-origins-iframe.htm:340]: Same issue here

File: oie/ja-jp/content/topics/api/trusted-origins-iframe.htm [hunk 3]
```diff
@@ -361,7 +361,7 @@ <h3><a name="2."></a>2. Okta End-User DashboardをiFrameに埋め込む</h3>
                                                     <h3><a name="3."></a>3. ［Customizations（カスタマイズ）］のiFrameから信頼済みオリジンに移行する</h3>
                                                     <ol>
                                                         <li value="1">
-                                                            <p><span class="uicontrol">［Customizations（カスタマイズ）］</span>&gt;<span class="uicontrol">［Other（その他）］</span>&gt;<span class="uicontrol">［Enable iFrame Embedding（iFrameの埋め込みを有効化）］</span>で構成されたすべてのiFrameを特定します。</p>
+                                                            <p><span class="uicontrol">［Customizations（カスタマイズ）］</span>&gt;<span class="uicontrol">［その他］</span>&gt;<span class="uicontrol">［Enable iFrame Embedding（iFrameの埋め込みを有効化）］</span>で構成されたすべてのiFrameを特定します。</p>
```
Comment (paulwallace-okta) [inline oie/ja-jp/content/topics/api/trusted-origins-iframe.htm:364]: Same issue here

File: oie/ja-jp/content/topics/api/trusted-origins-iframe.htm [hunk 4]
```diff
@@ -376,7 +376,7 @@ <h3><a name="3."></a>3. ［Customizations（カスタマイズ）］のiFrameか
                                                             <p>CSP違反を修正します。</p>
                                                         </li>
                                                         <li value="5">
-                                                            <p>すべての信頼済みオリジンが適切に機能していることを確認したら、<span class="uicontrol">［Customizations（カスタマイズ）］</span>&gt;<span class="uicontrol">［Other（その他）］</span>&gt;<span class="uicontrol">［Enable iFrame Embedding（iFrameの埋め込みを有効化）］</span>でiFrameの埋め込みのチェックを外します。これにより、すべての信頼済みオリジンが有効になります。</p>
+                                                            <p>すべての信頼済みオリジンが適切に機能していることを確認したら、<span class="uicontrol">［Customizations（カスタマイズ）］</span>&gt;<span class="uicontrol">［その他］</span>&gt;<span class="uicontrol">［Enable iFrame Embedding（iFrameの埋め込みを有効化）］</span>でiFrameの埋め込みのチェックを外します。これにより、すべての信頼済みオリジンが有効になります。</p>
```
Comment (paulwallace-okta) [inline oie/ja-jp/content/topics/api/trusted-origins-iframe.htm:379]: Same issue here

File: oie/ja-jp/content/topics/apps/aiw-saml-reference.htm [hunk 5]
```diff
@@ -413,15 +413,15 @@ <h2><a name="Task3"></a>SAML設定</h2>
                                                                 <td class="TableStyle-standard-horizonal-borders-only-BodyD-Column1-Body1">SAMLアサーションの暗号化に使用する公開鍵証明書（PEMフォーマット）を含むファイル。<p>このフィールドは、<span class="uicontrol">［Assertion Encryption（アサーション暗号化）］</span>が<span class="uicontrol">［Encrypted（暗号化済み）］</span>に設定されている場合に表示されます。</p></td>
                                                             </tr>
                                                             <tr class="TableStyle-standard-horizonal-borders-only-Body-Body1">
-                                                                <td class="TableStyle-standard-horizonal-borders-only-BodyE-Column1-Body1"><span class="uicontrol">［Signature Certificate（署名証明書）］</span>
+                                                                <td class="TableStyle-standard-horizonal-borders-only-BodyE-Column1-Body1"><span class="uicontrol">［署名証明書］</span>
```
Comment (paulwallace-okta) [inline oie/ja-jp/content/topics/apps/aiw-saml-reference.htm:416]: EN dropped from UI control

File: oie/ja-jp/content/topics/provisioning/azure/haad-join/prereqs-haad.htm [hunk 6]
```diff
@@ -381,10 +381,10 @@ <h2><a name="Prerequi2"></a>Okta環境の前提条件</h2>
                                                                     <p>シングルサインオンの構成時に、<span class="uicontrol">WS-Federation（自動または手動）</span>の方法を選択します。</p>
                                                                 </li>
                                                                 <li>
-                                                                    <p>プロビジョニングにOktaを使用している場合は、<span class="uicontrol">プロビジョニングタイプ</span>を<span class="uicontrol">Licenses/Role Management Only</span>または<span class="uicontrol">Profile Sync</span>に設定します。ほかのオプション（［User Sync（ユーザーの同期）］と［Universal Sync（ユニバーサル同期）］）は、ハイブリッドのセットアップに必要なAzure AD Connectと互換性がありません。</p>
+                                                                    <p>プロビジョニングにOktaを使用している場合は、<span class="uicontrol">プロビジョニングタイプ</span>を<span class="uicontrol">Licenses/Role Management Only</span>または<span class="uicontrol">Profile Sync</span>に設定します。ほかのオプション（［ユーザー同期］と［ユニバーサル同期］）は、ハイブリッドのセットアップに必要なAzure AD Connectと互換性がありません。</p>
```
Comment (paulwallace-okta) [inline oie/ja-jp/content/topics/provisioning/azure/haad-join/prereqs-haad.htm:384]: Here's another one

File: oie/ja-jp/content/topics/provisioning/azure/haad-join/prereqs-haad.htm [hunk 7]
```diff
@@ -381,10 +381,10 @@ <h2><a name="Prerequi2"></a>Okta環境の前提条件</h2>
...
                                                                 </li>
                                                                 <li>
-                                                                    <p>プロビジョニング中にプロファイルマッピングを構成する際に、ユーザーがデバイスへのサインインに使用するユーザー名と一致するように<span class="uicontrol">アプリケーションユーザー名のフォーマット</span>フィールドを設定します。通常、これはAzure ADユーザープリンシパル名です。</p>
+                                                                    <p>プロビジョニング中にプロファイルマッピングを構成する際に、ユーザーがデバイスへのサインインに使用するユーザー名と一致するように<span class="uicontrol">［アプリケーションユーザー名のフォーマット］</span>フィールドを設定します。通常、これはAzure ADユーザープリンシパル名です。</p>
```
Comment (paulwallace-okta) [inline oie/ja-jp/content/topics/provisioning/azure/haad-join/prereqs-haad.htm:387]: Not a regression, this is missing the EN portion already. We have many of these, it now seems.

**Review by emikhasyak-okta — APPROVED** [review]
🚀

**Pull request merged**